apps/agreement/contracts/disputable/DisputableApp.sol

```diff
@@ -0,0 +1,177 @@
+
+    // bytes32 internal constant AGREEMENT_POSITION = keccak256("aragonOS.appStorage.agreement");
+    bytes32 internal constant AGREEMENT_POSITION = 0x6dbe80ccdeafbf5f3fff5738b224414f85e9370da36f61bf21c65159df7409e9;
+
+    event AgreementSet(IAgreement indexed agreement);
+
+    modifier onlyAgreement() {
+        require(address(_getAgreement()) == msg.sender, ERROR_SENDER_NOT_AGREEMENT);
+        _;
+    }
+
+    /**
+    * @notice Set disputable agreements to `_agreement`
+    * @param _agreement Agreement instance to be linked
+    */
+    function setAgreement(IAgreement _agreement) external auth(SET_AGREEMENT_ROLE) {
+        IAgreement currentAgreement = _getAgreement();
+        bool settingNewAgreement = currentAgreement == IAgreement(0) && _agreement != IAgreement(0);
+        bool unsettingAgreement = currentAgreement != IAgreement(0) && _agreement == IAgreement(0);
+        require(settingNewAgreement || unsettingAgreement, ERROR_AGREEMENT_ALREADY_SET);
+
+        AGREEMENT_POSITION.setStorageAddress(address(_agreement));
+        emit AgreementSet(_agreement);
+    }
+
+    /**
+    * @notice Challenge disputable #`_disputableId`
+    * @param _disputableId Identification number of the disputable to be challenged
+    * @param _challenger Address challenging the disputable
+    */
+    function onDisputableChallenged(uint256 _disputableId, address _challenger) external onlyAgreement {
+        _onDisputableChallenged(_disputableId, _challenger);
+    }
+
+    /**
+    * @notice Allow disputable #`_disputableId`
+    * @param _disputableId Identification number of the disputable to be allowed
+    */
+    function onDisputableAllowed(uint256 _disputableId) external onlyAgreement {
+        _onDisputableAllowed(_disputableId);
+    }
+
+    /**
+    * @notice Reject disputable #`_disputableId`
+    * @param _disputableId Identification number of the disputable to be rejected
+    */
+    function onDisputableRejected(uint256 _disputableId) external onlyAgreement {
+        _onDisputableRejected(_disputableId);
+    }
+
+    /**
+    * @notice Void disputable #`_disputableId`
+    * @param _disputableId Identification number of the disputable to be voided
+    */
+    function onDisputableVoided(uint256 _disputableId) external onlyAgreement {
+        _onDisputableVoided(_disputableId);
+    }
+
+    /**
+    * @notice Tells whether the Agreement app is a forwarder or not
+    * @dev IForwarder interface conformance
+    * @return Always true
+    */
+    function isForwarder() external pure returns (bool) {
+        return true;
+    }
+
+    /**
+    * @dev Tell the agreement linked to the disputable instance
+    * @return Agreement linked to the disputable instance
+    */
+    function getAgreement() external view returns (IAgreement) {
+        return _getAgreement();
+    }
+
+    /**
+    * @notice Create a new action
+    * @dev IForwarder interface conformance
+    */
+    function forward(bytes) public;
```

Comment:
Do we need to declare this here, taking into account that it is already `IForwarder`?